app/actions/index.js

```diff
@@ -191,3 +203,12 @@ export const dropFolder = folder => async dispatch => {
   if (!isDirectory) return
   addDat({ path: folder.path })(dispatch)
 }
+
+export const loadFromLocalStorage = () => dispatch => {
+  const blob = localStorage.getItem('state')
+  if (!blob) return
+  const dats = JSON.parse(blob)
+  for (const dat of dats) {
+    addDat(dat)(dispatch)
```

Comment:
adding to my former point: This should be called something other than dat  in order to have structure <-> name consistency. `datPath`?